scalameta/parsers/js/src/main/scala/scala/meta/parsers/JSFacade.scala

```diff
@@ -108,14 +106,11 @@ object JSFacade {
         }
     }
 
-  @JSExportTopLevel("default")
-  @JSExportTopLevel("parseSource")
   def parseSource(
```

Comment:
Are these methods still exported to JS? This API is published to NPM as https://www.npmjs.com/package/scalameta-parsers

Reply:
I do not think these are no longer exported without annotation.

I will try to update so these APIs are exported.

Reply:
If `@JSExportTopLevel` annotations exists, the following error produed by Scala.js: 
```
[error] file:/home/vagrant/IdeaProjects/scalameta/scalameta/parsers/js/src/main/scala/scala/meta/parsers/JSFacade.scala(39:7:This): this of type <notype> typed as scala.meta.parsers.JSFacade$
```

I will look into.

Reply:
No idea how to fix this 🤷 ...
[Opened a question in Scala.js gitter](https://gitter.im/scala-js/scala-js?at=5ebf9003863019312a61f4a5).

Reply:
Filed an issue

Reply:
Looks like the solution is to use method overloads instead of default parameters. That's OK with me 👍

Reply:
Added overloads to fix the issue.